app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -71,12 +80,28 @@ import com.mapbox.maps.viewannotation.annotatedLayerFeature
 import com.mapbox.maps.viewannotation.annotationAnchor
 import com.mapbox.maps.viewannotation.geometry
 import com.mapbox.maps.viewannotation.viewAnnotationOptions
+import com.mapbox.turf.TurfMeasurement
 
 const val defaultZoom = 16.0
 const val maxZoom = 18.0
 const val minZoom = 8.0
 const val thresholdDisplayZoom = 13.0
 const val LAYER_ID = "0128"
+const val LABEL_THRESHOLD = 15.5
```

Comment:
In my opinion this name is not very clear. Maybe something like "Advanced mode zoom treshold" would be clearer?

Reply:
Fixed